### ISSUE TITLE
Question about model prediction results

### ISSUE THREAD

**Issue by Liu990406** [openclimatefix/skillful_nowcasting#56]
Thank you very for your work. I normalized the data in the preprocessing stage and denormalized the model prediction results to get the real prediction situation. I trained for a certain number of epochs(500), but I still cannot eliminate the grid lines that appear in the model prediction results, as shown in the picture below. I wonder if you have any relevant experience.
![DGMR_predict](https://github.com/openclimatefix/skillful_nowcasting/assets/135589550/0a82d492-2375-4118-875a-878cc06c641e)

I appreciate your attention and assistance.

**Comment by peterdudfield** [comment]
@jacobbieker was this something to do with the loss function used? I've definitely seen this before
